README.md

```diff
@@ -29,7 +29,7 @@ To test the RNG, we just pipe the output of the script to test
 `testu01_gateway` executable in the docker image:
 
 ```bash
-python3 test_rng_example.py | docker run -it blep/TESTU01_gateway:latest --smallcrunch
+python3 test_rng_example.py | docker run --rm -it blep/test01:latest --small-crush
```

Comment:
I think it would be even better if there was one step (build docker image)  before this instruction. Then, one could use the correct image name and only had to copy paste these instructions.